airflow/providers/yandex/hooks/yandex.py

```diff
@@ -85,17 +85,24 @@ def get_connection_form_widgets() -> dict[str, Any]:
             ),
         }
 
+    @classmethod
+    def airflow_user_agent(cls) -> str:
+        import airflow
+        import os
+
+        version = os.getenv("AIRFLOW_USER_AGENT_VERSION", airflow.__version__)
```

Comment:
Maybe better make it as a part of Provider Configuration?
- [Setting Configuration Options](https://airflow.apache.org/docs/apache-airflow/stable/howto/set-config.html)
- [Global Configuration Reference](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#configuration-reference)
- [OpenLinage Configuration Reference](https://airflow.apache.org/docs/apache-airflow-providers-openlineage/stable/configurations-ref.html)